source/mainnet/smart-contracts/tutorials/nft-minting/build-smart-contract.rst

```diff
@@ -35,21 +35,19 @@ If everything is correct, the ``init`` command will show something like the belo
 Build your smart contract
 =========================
 
-Now you are ready to build your contract. Create a ``dist/cis2-nft`` folder to keep your Wasm compiled output file and schema file first and then build it with the following command.
+Now you are ready to build your contract. First, open up your lib.rs file under /src. You need to specify the ``TOKEN_METADATA_BASE_URL`` parameter in the contract. Since this is an NFT, you need to store the metadata on chain forever. Add your IPFS link like below.
```

Comment:
The metadata is usually large (e.g. PNG files), which is not saved on chain but instead a small URL to that metadata is stored on chain.

```suggestion
Now you are ready to build your contract. First, open up your ``lib.rs`` file under ``/src``. You need to specify the ``TOKEN_METADATA_BASE_URL`` parameter in the contract. Since this is an NFT, you store a URL to its metadata on chain. Add your IPFS link like below.
```